scripts/console.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+ARGS=$*
+
+echo "Run the RWD Docker container."
+
+if [ -z "$RWD_DATA" ]; then
+    echo "Environment variable RWD_DATA not defined."
+    exit 1
+fi
+
+set -x
+
+docker-compose run \
+--rm \
+-e RWD_DATA=$RWD_DATA \
+--entrypoint /bin/bash \
+app
+$ARGS
```

Comment:
Probably more bugs from the existing version of this script but I noticed 2 things:
* Lines 17-21 should be indented
* Line 20 is missing a trailing slash
* Passing $ARGS to bash in this context is really unnecessary